### ISSUE TITLE
elasticsearch-sql as a standalone library

### ISSUE THREAD

**Issue by Jarlakxen** [NLPchina/elasticsearch-sql#67]
It could be really nice if we could be able to use the sql to es conversion as a standalone library ( Like https://github.com/drewbo/sql-to-es )

**Comment by omershelef** [comment]
I agree, for now see issue #58. It possible to use elasticsearch-sql without installiing it.

**Comment by dmfenton** [comment]
:+1: This would be awesome. I'm currently considering installing this package just to use the explain functionality.
